source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/core-metrics-results/faith_pd_vector.qza
+   :saveas: faith_pd_vector.qza
+
+To view this artifact as metadata, simply pass it in to any method or visualizer that expects to see metadata (e.g. ``metadata tabulate``):
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file faith_pd_vector.qza
+     --o-visualization tabulated-faith-pd-md.qzv
+
+.. question::
+   What is the largest value of Faith's PD? What is the smallest? Hint: use the column sorting functions to assist with this query.
+
+Combining metadata
+------------------
+
+Because metadata can come from many different sources, QIIME 2 supports metadata merging when running commands. Building upon the examples above, simply passing ``--m-input-file`` multiple times will combine the metadata in the specified files:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv
+     --m-input-file faith_pd_vector.qza
+     --o-visualization tabulated-combined-md.qzv
+
+Merging is based on the identifiers specified in the first column of each metadata file (e.g. Sample IDs or Feature IDs). The resulting metadata after the merge represents the intersection of the identifiers in all of the specified files. Note, merging is non-destructive, and does not modify any of the input files.
+
+.. question::
+   Using another alpha diversity QZA, what happens when merging three files? How many columns are present in the resulting metadata file? How many of those columns represent the Sample IDs? How many of those columns represent alpha diversity metrics?
+
+Metadata merging is supported anywhere that metadata is accepted within the QIIME 2 ecosystem. For example, it might be interesting to consider coloring an Emperor plot based on the study metadata, or the results of one of the study's alpha diversity metrics. This can be accomplished by providing both the sample metadata mapping file *and* the Faith's PD QZA:
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/core-metrics-results/unweighted_unifrac_pcoa_results.qza
+   :saveas: unweighted_unifrac_pcoa_results.qza
+
+.. command-block::
+   qiime emperor plot \
+     --i-pcoa unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-file faith_pd_vector.qza \
+     --o-visualization unweighted-unifrac-emperor-with-alpha.qzv
+
+.. question::
+   What happens to the visualization if the order of the metadata files is reversed in the above ``emperor plot``? What about in the ``metadata tabulate`` example above?
+
+Exploring feature metadata
+--------------------------
+
+As noted above, metadata can be viewed in the context of one of two axes: sample-specific or feature-specific. The previous examples in this tutorial were focused on sample-oriented metadata.
```

Comment:
*metadata can be viewed in the context of one of two axes: sample-specific or feature-specific* -> *metadata can exist for either axis of a `FeatureTable`: the sample axis or the feature axis*